features/example_groups/basic_structure.feature

```diff
@@ -38,6 +38,7 @@ Feature: basic structure (`describe`/`it`)
         it "does one thing" do
         end
       end
+      
```

Comment:
The build is failing because this line has spaces.  We enforce that blank lines should be only a new line with no other characters.  If you fix that, the build should go green and we can merge this.